[2025-02-21T16:38:50.388+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-21T16:38:29.131770+00:00 [queued]>
[2025-02-21T16:38:50.482+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-21T16:38:29.131770+00:00 [queued]>
[2025-02-21T16:38:50.482+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-21T16:38:50.564+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): kafka_data_stream> on 2025-02-21 16:38:29.131770+00:00
[2025-02-21T16:38:50.585+0000] {standard_task_runner.py:57} INFO - Started process 21034 to run task
[2025-02-21T16:38:50.734+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'kafka_spark_dag', 'kafka_data_stream', 'manual__2025-02-21T16:38:29.131770+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/dag_kafka_spark.py', '--cfg-path', '/tmp/tmpahuzauy0']
[2025-02-21T16:38:50.795+0000] {standard_task_runner.py:85} INFO - Job 64: Subtask kafka_data_stream
[2025-02-21T16:38:51.759+0000] {task_command.py:416} INFO - Running <TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-21T16:38:29.131770+00:00 [running]> on host c39f67883fe1
[2025-02-21T16:38:52.601+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='kafka_spark_dag' AIRFLOW_CTX_TASK_ID='kafka_data_stream' AIRFLOW_CTX_EXECUTION_DATE='2025-02-21T16:38:29.131770+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-21T16:38:29.131770+00:00'
[2025-02-21T16:39:05.986+0000] {logging_mixin.py:154} INFO - Dataset downloaded successfully.
[2025-02-21T16:39:06.338+0000] {logging_mixin.py:154} INFO - Dataset extracted successfully.
[2025-02-21T16:39:06.339+0000] {logging_mixin.py:154} INFO - No CSV file found in extracted data.
[2025-02-21T16:39:06.342+0000] {logging_mixin.py:154} INFO - Dataset preparation failed.
[2025-02-21T16:39:06.346+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-21T16:39:06.502+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=kafka_spark_dag, task_id=kafka_data_stream, execution_date=20250221T163829, start_date=20250221T163850, end_date=20250221T163906
[2025-02-21T16:39:07.071+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-21T16:39:07.493+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
